gems/pending/spec/miq_vm/miq_azure_vm_image_spec.rb

```diff
@@ -12,8 +12,10 @@
     @client_id  = @test_env[:azure_client_id]
     @client_key = @test_env[:azure_client_key]
     @tenant_id  = @test_env[:azure_tenant_id]
-    @image_name = @test_env[:image_name]
-    @image_uri  = @test_env[:image_uri]
+    @image_name = "https://miqazuretest14047.blob.core.windows.net/system/Microsoft.Compute/" \
+                  "Images/miq-test-container/test-win2k12-img-osDisk.e17a95b0-f4fb-4196-93c5-0c8be7d5c536.vhd"
+    @image_uri  = "https://miqazuretest14047.blob.core.windows.net/system/Microsoft.Compute/" \
+                  "Images/miq-test-container/test-win2k12-img-osDisk.e17a95b0-f4fb-4196-93c5-0c8be7d5c536.vhd"
```

Comment:
@bronaghs this shouldn't change, it's handled automatically by the `TestEnvHelper`